packages/pigeon/platform_tests/ios_unit_tests/ios/Runner/async_handlers.m

```diff
@@ -0,0 +1,86 @@
+// Autogenerated from Pigeon (v0.1.19), do not edit directly.
+// See also: https://pub.dev/packages/pigeon
+#import "async_handlers.h"
+#import <Flutter/Flutter.h>
```

Comment:
And then this can be removed when it's imported in the header.

Reply:
ack, see above